pkg/descriptions.json

```diff
@@ -1962,6 +1962,13 @@
       "System.Text.Encodings.Web.JavaScriptEncoder"
     ]
   },
+  {
+    "Name": "System.Text.Json",
+    "Description": "Low allocations UTF-8 JSON reader/writer and serializer/deserializer.",
```

Comment:
I'm wondering if we should only mention reader/writer in this PR and then update it to include serializer/deserializer when they're actually there.  I'm fine either way.